playgrounds/esm/tsconfig.node20.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
-    "module": "Node16",
-    "moduleResolution": "Node16"
+    "module": "Node20",
+    "moduleResolution": "NodeNext"
```

Comment:
These changes are not required-since ESM will continue running on the TS Node16 setup. I'd suggest of simply adding module target of node 20 as a separate configuration (and the according scripts of course). 

Keeping this option (at least for ESM) give us safety of getting alerted when we break something for users with these setting. We can then assert if that change is worth actually breaking our users setup or not.

As pointed out in the last meeting a lot of users probably do not set these fields themself, but rely on meta frameworks to orchestrate them instead. Simply saying set these fields to value "x" might not work for them.

Reply:
I set them in esm, as I thought we don't support node<=18 anymore

Reply:
We have these setups to inform us that specific project setups should work with Faker. That does not mean that we support these versions. Regardless, we gain valuable information from these setups and can assure that they continue to work over multiple versions of Faker - or at least get informed if on of these setup would break so we can inform our users.